experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+.columns {
+ display: flex;
+}
+
+.drag-drop {
+  display: inline-block;
+  position: absolute;
+  top: 0px;
+  width: 50px;
+  height: 50px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  transform: translate(0px, 0px);

```

Comment:
Why do we need the transform property when we are not doing any transformation?

Reply:
Yes, it was unnecessary, removed it.